docs/src/increase-observability/logs/forward-logs.md

```diff
@@ -110,20 +110,20 @@ You can forward your Platform.sh logs to any of those endpoints.
 To enable this forwarding, run a command like the following:
 
 ```bash
-platform integration:add --type syslog --host {{< variable "INGESTION_HOST" >}} --port {{< variable "INGESTION_HOST_PORT" >}}
+platform integration:add --type syslog --syslog-host {{< variable "INGESTION_HOST" >}} --syslog-port {{< variable "INGESTION_HOST_PORT" >}}
 ```
 
-`type`, `host`, and `port` are the only properties required for all endpoints.
+`type`, `syslog-host`, and `syslog-port` are the only properties required for all endpoints.
 The following table shows the other available properties:
 
-| Property         | Type      | Default    | Description |
-| ---------------- | --------- | ---------- | ----------- |
-| `auth_token`     | `string`  |            | The token to authenticate with the given service. |
-| `auth_mode`      | `string`  |            | The mode for authentication with the given service. Can be `prefix` or `structured_data`. |
+| Property         | Type      | Default    | Description                                                                                                                                           |
+| ---------------- | --------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `auth-token`     | `string`  |            | The token to authenticate with the given service.                                                                                                     |
+| `auth-mode`      | `string`  |            | The mode for authentication with the given service. Can be `prefix` or `structured_data`.                                                             |
 | `facility`       | `string`  | `1` (user) | A [syslog facility code](https://en.wikipedia.org/wiki/Syslog#Facility) to attach with each log to identify the source. Can be a number from 0 to 23. |
-| `message_format` | `string`  | `rfc5424`  | The standard to use for the message format. Can be `rfc5424` or `rfc3164`. |
-| `protocol`       | `string`  | `tls`      | The network protocol to use in the connection. Can be one of `tls`, `tcp`, or `udp`. Defaults to `tls`. |
```

Comment:
Are you sure about this? At least the help text indicates `tls` is still the default.

Reply:
Seems to be: https://github.com/platformsh/legacy-cli/blob/main/src/Command/Integration/IntegrationCommandBase.php#L511

Reply:
you're right, I was too quick